pythonz/downloader.py

```diff
@@ -79,3 +81,13 @@ def fetch(cls, url, filename):
         except IOError:
             sys.stdout.write('\n')
             raise DownloadError('Failed to fetch %s from %s' % (filename, url))
+
+
+def validate_sha256(filename, sha256sum):
+    if sha256sum is not None:
+        with open(filename, 'rb') as f, mmap(f.fileno(), 0, access=ACCESS_READ) as m:
```

Comment:
Uhm, you're probably using it with an old Python version.
Can't remember when mmap has been made a context manager... The workaround for old pythons is to wrap it in `contextlib.closing`
-- 
Sent from mobile. Please excuse my brevity.